save-frontend/src/main/kotlin/com/saveourtool/save/frontend/App.kt

```diff
@@ -262,16 +269,20 @@ class App : ComponentWithScope<PropsWithChildren, AppState>() {
                                     element = organizationView.create()
                                 }
 
-                                Route {
-                                    path = "/:owner/:name"
-                                    element = projectView.create()
-                                }
+                                createRoutersWithPathAndEachListItem(OrganizationMenuBar.listOfStringEnumElements, "/organization/:owner", organizationView.create())
```

Comment:
this is not readable for me, frankly speaking

Reply:
Fixed